main/WebServer.cpp

```diff
@@ -1323,7 +1354,7 @@ namespace http {
 
 				root["idx"] = sd[0].c_str(); // OTO output the created ID for easier management on the caller side (if automated)
 
-				m_mainworker.AddHardwareFromParams(ID, name, (senabled == "true") ? true : false, htype, address, port, sport, username, password, extra, mode1, mode2, mode3, mode4, mode5, mode6, iDataTimeout, true);
+				m_mainworker.AddHardwareFromParams(ID, name, (senabled == "true") ? true : false, htype, address, port, sport, username, password, extra, mode1, mode2, mode3, mode4, mode5, mode6, mode1Str, mode2Str, mode3Str, mode4Str, mode5Str, mode6Str, iDataTimeout, true);
```

Comment:
I think the 'extra' field should be sufficient for all hardware
If you need multiple extra parameters, you could contact them by a divider